composer.json

```diff
@@ -9,7 +9,8 @@
     "laravel/tinker": "^1.0",
     "aws/aws-sdk-php-laravel": "^3.1",
     "barryvdh/laravel-debugbar": "^2.3",
-    "doctrine/dbal": "^2.5",
+    "doctrine/dbal": "~2.5.13",
+    "doctrine/common": "~2.6.0",
```

Comment:
We added that when we were debugging the Composer weirdness. It's a dependency of `doctrine/dbal` so shouldn't be necessary here.